samples/demo/src/main/res/layout/activity_main.xml

```diff
@@ -67,6 +67,19 @@
         style="@style/label"
         />
     </FrameLayout>
+
+    <FrameLayout style="@style/sample_layout">
+      <com.facebook.drawee.view.SimpleDraweeView
+        android:id="@+id/attr_image_url"
+        style="@style/drawee"
+        fresco:imageUri="http://www.smemo.info/icon.png"
```

Comment:
Can you please just change this line so that the sample points to an image from our own domain. A sensible URI would be `http://frescolib.org/static/favicon.png`